humanmade/altis-enhanced-search
# ElasticSearch user dictionary support

Adds a network admin and site admin settings page to upload or manually enter a user dictionary, stop words and synonyms.

Still to do:

- [x] Test with local environments, docker & chassis
- [x] Test AWS Elasticsearch Service integration
- [x] Filter ES mapping to:
  - Add user dictionary to kuromoji tokenizer
  - Add stop words filters for manual & uploaded to default analyzer
  - Add synonyms filters for manual & uploaded to default analyzer
  - Support network wide override for sites with same language
- [x] Handle reindexing on update using `_update_by_query` ES endpoint

See #32.
